foundations/installations/installing_node.md

```diff
@@ -82,3 +87,112 @@ nvm use node
 Now when you run `node -v` you should see `v14.10.0` or something similar.
 
 If you see that, you have successfully installed node!
+
+
+# Installation on MacOS (zsh)
```

Comment:
This doesn’t follow my directions- we want to *include* zsh not replace the bash instructions.

Where did you get these instructions from? We really only needed to reference the two zsh files. We didn’t want to completely overhaul our directions.

Reply:
antigen is a package manager of zsh, doing things this way will be much easier for Mac users with zsh shell

Reply:
It’s out of scope, here.

Reply:
users of mac catalina or above will get into many permission issues later.
Many have complained about it, 
Using a package manager is a robust way accomplish the task.